### ISSUE TITLE
Name items - Special symbols support

### ISSUE THREAD

**Issue by truefriend-cz** [zemian/index-listing#4]
Do you can add support for browsing folders begin with special symbols (exclamation mark) in name?

**Comment by zemian** [comment]
No, sorry, the index.php script is simple enough that you can modify it to your own need. Feel free to fork it.

**Comment by truefriend-cz** [comment]
Yes, Unfortunately, although I am a programmer, I am in a human environment where I am prohibited, blocked from programming, editing and developing other new scripts. The local state police is completely useless because they don't consider it a thing for making money or any operational needs of a person. There are legalized forms of murder in my area where some members of the state police are also used. This has been happening for about nine years and has no restrictions or stops. Whether I'm dealing with other members of the police or government prosecutors, etc. I'm trying my best, but I have more work to do so than to be able to work on the development or editing of scripts, not only in PHP. I apologize for the longer post. And a mention of the problem. Maybe in time it will be better if I live to see it.

**Comment by zemian** [comment]
Sorry to hear your problem in life, but I can't help you there.

**Comment by truefriend-cz** [comment]
Yes I know. Many people tell me that. I hear it for an average of nine years every 2.5 hours. That's why I try to solve the script and not what's going on around me. A script solutions has maybe more solution options than others social solutions.
